tests/unit/systems/utils/triton.py

```diff
@@ -1,40 +0,0 @@
-#
-# Copyright (c) 2022, NVIDIA CORPORATION.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-from distutils.spawn import find_executable
-
-import pytest
-from tritonclient import grpc as grpcclient
-
-triton = pytest.importorskip("merlin.systems.triton")
-data_conversions = pytest.importorskip("merlin.systems.triton.conversions")
-
-TRITON_SERVER_PATH = find_executable("tritonserver")
-from merlin.systems.triton.utils import run_triton_server  # noqa
-
-
-def _run_ensemble_on_tritonserver(
```

Comment:
We've had two versions of this method kicking around for a while, and the public version is more convenient to use since it automatically converts the response to a dictionary.